docs/content/integrations/dbt/using-dbt-with-dagster/load-dbt-models.mdx

```diff
@@ -15,113 +15,146 @@ description: Dagster can orchestrate dbt alongside other technologies.
-```
+To build your dbt project, i.e. materialize your assets, click the **Materialize all** button near the top right corner of the page. This which will launch a run to materialize the assets. When finished, the **Materialized** and **Latest Run** attributes in the asset will be populated:
 
-Let's take a look at what's happening here:
+<Image
+alt="Asset graph in Dagster's UI, showing materialized assets"
+src="/images/integrations/dbt/using-dbt-with-dagster/upstream-assets-asset-graph-materialized.png"
+width={1091}
+height={896}
+/>
 
-- In the `resources` key, we've provided configuration info for the <PyObject module="dagster_dbt" object="DbtCli" /> resource.
-- Added all assets in the assets module and the resources mapped to the `resources` key to the <PyObject object="Definitions" /> object. This supplies the resource we created to our assets.
-- Using <PyObject object="load_assets_from_modules" />, we've added all assets in the `assets` module as definitions. This approach allows any new assets we created to be automatically included in the code location instead of needing to manually add them one by one.
+After the run completes, you can:
+
+- Click the **asset** to open a sidebar containing info about the asset, including its last materialization stats and a link to view the [**Asset details** page](/concepts/dagit/dagit#asset-details)
+- Click the ID of the **Latest Run** - in the above image, that's `651489a2` - in an asset to view the [**Run details**](/concepts/dagit/dagit#run-details) page. This page contains detailed info about the run, including timing information, errors, and logs.
 
 ---
 
-## Step 3: View the assets in the Dagster UI
+## Step 4: Review the Python code in your Dagster project
 
-In this step you'll start Dagster's web UI.
+You saw how you can create a Dagster project that loads a dbt project. How does this work? Understanding how Dagster loads a dbt project will give you a foundation for customizing how Dagster runs your dbt project, as well as for connecting it to other data assets outside of dbt.
 
-1. To start the UI, run the following in `/tutorial_template`:
+The most important file is the Python file that contains the set of definitions for Dagster to load: `jaffle_shop/definitions.py`. Dagster executes the code in this file to find out what assets it should be aware of, as well as details about those assets. For example, when you ran `dagster dev` in the previous step, Dagster executed the code in this file to determine what assets to display in the UI.
 
-   ```shell
-   dagster dev
-   ```
+In order to return a Dagster asset for each dbt model, the code in this file needs to find out what dbt models you have. It finds that out by reading a file called a "manifest.json", which is a file that dbt can generate for any dbt project and contains information about every model, seed, snapshot, test, etc. in the project. Here's the code that does this:
```

Comment:
```suggestion
To return a Dagster asset for each dbt model, the code in this file needs to know what dbt models you have. This is accomplished by reading a file called a `manifest.json`, which is a file that dbt can generate for any dbt project and contains information about every model, seed, snapshot, test, etc. in the project. Here's the code that does this:
```

Suggesting some small tweaks to make this more concise.

Is the code that follows what's in the user's `definitions.py` already? If so, I think explicitly stating that (over `Here's the code that does this`) would be helpful. Ex: 

> The following code, already in your project's `definitions.py`, [does a thing]